vllm/sequence.py

```diff
@@ -139,6 +156,9 @@ def _append_logical_block(self) -> None:
             block_size=self.block_size,
         )
         self.logical_token_blocks.append(block)
+        
+    def _delete_logical_block(self, block: LogicalTokenBlock) -> None:
+        self.logical_token_blocks.remove(block)
```

Comment:
any reason to create this method instead of just inplace use `self.logical_token_blocks.remove(block)`?